packages/experimental/src/experimental-list/experimental-list-item.tsx

```diff
@@ -2,12 +2,19 @@
  * External dependencies
  */
 import { CSSTransition } from 'react-transition-group';
+import { ENTER } from '@wordpress/keycodes';
 import classnames from 'classnames';
 
-/**
- * Internal dependencies
- */
-import { handleKeyDown } from './list-item';
+function handleKeyDown(
+	event: React.KeyboardEvent< HTMLElement >,
+	onClick?:
+		| React.MouseEventHandler< HTMLElement >
+		| React.KeyboardEventHandler< HTMLElement >
+) {
+	if ( typeof onClick === 'function' && event.keyCode === ENTER ) {
+		( onClick as React.KeyboardEventHandler< HTMLElement > )( event );
+	}
+}
```

Comment:
I assume this was to avoid a dependency on `@woocommerce/components`?

Reply:
Yeah exactly.